src/_posts/databases/postgresql/2000-01-01-overview.md

```diff
@@ -58,11 +58,37 @@ The communication between the PostgreSQL® nodes is encrypted. Choosing such a
 plan drastically lowers the risk of unavailability, be it during a maintenance
 operation (database upgrade, for example) or because of a platform issue.
 
-### Available Version
+### Life-Cycle
 
-The latest and default version of PostgreSQL® available is **`15.8.0-1`**.
+The PostgreSQL® project releases new major versions annually, introducing new
+features and performance improvements. At Scalingo, we ensure compatibility
+with these major versions, typically making them available on our platform
+about one year after their initial release.
 
-Three last major versions are officially maintained: v15, v14 and v13.
+While the PostgreSQL® community supports a major version for up to five years
+after its initial release, Scalingo supports each major version on our platform
+for three years, providing minor upgrades that include critical bug fixes and
+security patches. After this period, we gradually deprecate oldest versions to
+ensure security, performance, and operational efficiency.
+
+#### Available Versions
+
+| PostgreSQL Version  | Full Version      |
+| ------------------: | ----------------: |
+| **`15`**            | up to `15.8.0-1`  |
+| `14`                | up to `14.13.0-2` |
+| `13`                | up to `13.16.0-2` |
+
+Currently, the supported major versions of PostgreSQL® on the Scalingo platform
+are `13`, `14`, and `15`.
+
+The default version when provisioning a Scalingo for PostgreSQL® addon is
+**`15.8.0-1`**.
+
+{% note %}
+Please get in touch with our Support team if you need to provision a Scalingo
+for PostgreSQL® addon with a specific version.
```

Comment:
Je le mettrais peut etre dans l'autre sens, pour avoir l'info au début.

"If you need to provision a specific version of the Scalingo for PostgreSQL® addon, please contact our Support team.